services/decoder/protocols/aave-v3/aave-v3.configs.ts

```diff
@@ -0,0 +1,12 @@
+import { type Configs } from "../../decoder.types";
+
+const configs: Configs = [
+    {
+        address: "0x87870bca3f3fd6335c3f4ce8392d69350b4fa4e2",
+        is_factory: false,
+        protocol_name: "aave-v3",
+        network: "eth-mainnet",
+    },
+];
```

Comment:
Add configs fore 2-3 more chains

Reply:
just added all supported chain for aave v3 except for gnosis, scroll and fantom.